docs/model_server_kfs_grpc_api.md

```diff
@@ -0,0 +1,65 @@
+# gRPC API {#ovms_docs_grpc_api}
```

Comment:
file name is kfs but the content about TFS API

Reply:
Added into MR by mistake - initially thought about having separate md for KServe